src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -7,8 +7,10 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Globalization;
```

Comment:
>Globalization [](start = 13, length = 13)

Why is this needed?

Reply:
For `CultureInfo`

---
In reply to: [457684561](https://github.com/dotnet/roslyn/pull/46110#discussion_r457684561) [](ancestors = 457684561)

Reply:
> For CultureInfo

I think we should be using binary comparison option instead and simply concatenating strings instead of using ```string.Format``, if concatenation cannot be avoided. 

---
In reply to: [457780357](https://github.com/dotnet/roslyn/pull/46110#discussion_r457780357) [](ancestors = 457780357,457684561)